doc/source/scalability/innodb_io.rst

```diff
@@ -54,17 +54,16 @@ The following values are allowed:
     use ``fsync()`` to flush both the data and log files.
 
   * ``O_SYNC``: 
-    use O_SYNC to open and flush the log files; use ``fsync()`` to flush the data files.
+    use O_SYNC to open and flush the log files; use ``fsync()`` to flush the data files. When this value is set :ref:`parallel_doublewrite_buffer` file is opened with ``O_SYNC`` flag. 
 
   * ``O_DIRECT``: 
-    use O_DIRECT to open the data files and fsync() system call to flush both the data and log files.
+    use O_DIRECT to open the data files and fsync() system call to flush both the data and log files. 
 
   * ``O_DIRECT_NO_FSYNC``:
-    use O_DIRECT to open the data files but don't use ``fsync()`` system call to flush both the data and log files. This option isn't suitable for *XFS* file system.
+    use O_DIRECT to open the data files but don't use ``fsync()`` system call to flush both the data and log files. When this value is set, the :ref:`parallel_doublewrite_buffer` file is not flushed after a batch of writes to it is completed. This option isn't suitable for *XFS* file system.
```

Comment:
use O_DIRECT to open the data files, but don't use fsync() system call to flush data, log, and parallel doublewrite files.